### DISCUSSION TITLE
Calculate difference lowest and highest price

### DISCUSSION THREAD

**Discussion by dancekid** [mampfes/ha_epex_spot#151] [Q&A]
Good Evening,
i created an automation:
If 2 hour Lowest price then load battery.
Now i want to add:
Start only  if highest price - lowest price > 8 Cent

How to calculate the difference?

Greetings Michael

**Comment by aichingerk** [comment]
Hi, make a helper template binary sensor like:

```
 {% set Max = states('sensor.epex_spot_at_highest_price') | float  %}
  {% set Min = states('sensor.epex_spot_at_lowest_price') | float %}
  {% set Ergebnis = Max - Min %} 
  {% if (Ergebnis < 0.08) -%} 
    {{ false }}
  {%- else -%}
    {{ true }}
  {%- endif %}
```

And in your automation you can ask with a condition if your sensor is true .
